main.py

```diff
@@ -101,12 +100,13 @@ def create_transaction(self) -> None:
 
 
     def block_creation(self):
+        # WIP: use hash of 2 or 3 previous block as seed
+        random.seed(len(self.blocks))
```

Comment:
but what if some peers have different amount of blocks at this time? can happen

Reply:
yes, you are right, it can happen.
but if we use a constant number every peer will always chose the same mid, so every block will always be built by the same peer.
I committed a fix in the known_peers_mid list. This list should be the same to every peer, so I sorted it to achieve this.